### DISCUSSION TITLE
example for translated `opengraph-image.tsx`

### DISCUSSION THREAD

**Discussion by stefanprobst** [amannn/next-intl#466] [Q&A] [answered]
hi, does anyone have an example for translated `opengraph-image.tsx` metadata routes they could link to? thanks!

**Comment by amannn** [comment, marked as answer]
Did you see the docs on the topic? https://next-intl-docs.vercel.app/docs/environments/metadata-route-handlers#metadata-files

**Reply by stefanprobst** [reply]
ah thanks! so it seems my problem is caused by `export const runtime = "edge"`, which may be this upstream issue: https://github.com/vercel/next.js/issues/48295
